scripts/run-integration-tests-npm.sh

```diff
@@ -161,8 +173,54 @@ function setup_node() {
   cd "${cwd}"
 }
 
+function create_npm_package() {
+  local lib="${1}"
+  yarn workspace @hoprnet/${lib} pack
+  mv ${mydir}/../packages/${lib#hopr-}/package.tgz "${tmp}/${lib}-package.tgz"
+}
+
+function install_npm_packages() {
+  if [ -d "${npm_install_dir}" ]; then
+    # noop. already setup
+    :
+  else
+    # move into work dir before we proceed
+    mkdir -p "${npm_install_dir}"
+    cd "${npm_install_dir}"
+
+    if [ -n "${npm_package_version}" ]; then
+      npm install @hoprnet/hoprd@${npm_package_version}
+    else
+      npm install ${tmp}/hoprd-package.tgz
+      npm install ${tmp}/hopr-ethereum-package.tgz
+      npm install ${tmp}/hopr-core-ethereum-package.tgz
+      npm install ${tmp}/hopr-core-package.tgz
+      npm install ${tmp}/hopr-utils-package.tgz
+    fi
+
+    # Copies local deployment information to npm install directory
+    # Fixme: copy also other environments
+    # need to mirror contract data because of hardhat-deploy node only writing to localhost
```

Comment:
can we open an issue for this before we merge this?

Reply:
in our repo or hardhat/hardhat-deploy?

Reply:
the `FIXME` me part, which I think its for our repo

Reply:
ah I see, I think that one doesn't apply and can be removed even